<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddPrintWidget</class>
 <widget class="QWidget" name="AddPrintWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>431</width>
    <height>186</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="PrinterNameLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Наименование устройства:</string>
   </property>
  </widget>
  <widget class="QLabel" name="PrinterTypeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Тип&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="PrinterSiteLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>50</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Сайт&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddBut">
   <property name="geometry">
    <rect>
     <x>324</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackBut">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Назад</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>50</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PhotoTextEdit">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>90</y>
     <width>281</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="PrintNameText">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>9</y>
     <width>191</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="TypeComboBox">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="AddPhotoBut">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать фото</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
